build.gradle

```diff
@@ -35,30 +27,51 @@ sourceSets {
     }
     test {
         resources {
-            srcDirs = [ "src/test/java" , "src/test/resources"]
+            srcDirs = ["src/test/java", "src/test/resources"]
             exclude "**/*.java"
         }
     }
 }
 
 dependencies {
-    compile group: 'com.fasterxml.jackson.core', name: 'jackson-databind', version: '2.10.1' // Apache License 2.0
-    compile group: 'com.google.code.gson', name: 'gson', version: '2.8.6' // Apache License 2.0
-    compile group: 'ch.qos.logback', name: 'logback-classic', version: '1.2.3' // LGPL v2.1
-    compile group: 'io.swagger.parser.v3', name: 'swagger-parser', version: '2.0.16' // Apache License 2.0
-    compile group: 'info.picocli', name: 'picocli', version: '4.1.0' // Apache License 2.0
-    compile group: 'org.apache.httpcomponents', name: 'httpclient', version: '4.5.10' // Apache License 2.0
-    compile group: 'org.apache.httpcomponents', name: 'httpmime', version: '4.5.10' // Apache License 2.0
-    compile group: 'org.testng', name: 'testng', version: '7.0.0' // Apache License 2.0
-    compile group: 'org.slf4j', name: 'jcl-over-slf4j', version: '1.7.29' // MIT License (compatible with Apache License)
-    compile group: 'org.slf4j', name: 'slf4j-api', version: '1.7.29' // MIT License (compatible with Apache License)
+    implementation group: 'com.fasterxml.jackson.core', name: 'jackson-databind', version: '2.10.1'
+    // Apache License 2.0
+    implementation group: 'com.google.code.gson', name: 'gson', version: '2.8.6' // Apache License 2.0
+    implementation group: 'ch.qos.logback', name: 'logback-classic', version: '1.2.3' // LGPL v2.1
+    implementation group: 'io.swagger.parser.v3', name: 'swagger-parser', version: '2.0.16' // Apache License 2.0
+    implementation group: 'info.picocli', name: 'picocli', version: '4.1.0' // Apache License 2.0
+    implementation group: 'org.apache.httpcomponents', name: 'httpclient', version: '4.5.10' // Apache License 2.0
+    implementation group: 'org.apache.httpcomponents', name: 'httpmime', version: '4.5.10' // Apache License 2.0
+    implementation group: 'org.testng', name: 'testng', version: '7.0.0' // Apache License 2.0
+    implementation group: 'org.slf4j', name: 'jcl-over-slf4j', version: '1.7.29'
+    // MIT License (compatible with Apache License)
+    implementation group: 'org.slf4j', name: 'slf4j-api', version: '1.7.29'
+    // MIT License (compatible with Apache License)
 }
 
 configurations {
-    compile.exclude module:'commons-logging'
+    compile.exclude module: 'commons-logging'
 }
 
-findbugsMain.enabled = false
+jar {
```

Comment:
Please explain the reason behind this change. It was an executable jar before this change as well.

Reply:
Not really, it was a jar file with a manifest which pointed to jar files on the local file system. You would always need to compile it first, with this setup you can release a jar file which people can directly download without needing to build it, just type `java -jar tool.jar` from the command line. All the necessary classes are bundled in one jar

Reply:
Indeed. I didn't plan on releasing the jar as a standalone, but rather the people to incorporate this repo in their CI/CD process. This will bloat the jar file size drastically. Do you plan to consume the jar instead?

Reply:
But then people need to grab all the dependent jar themselves, leading to exact the same amount of copying all those jars around in the CI/CD pipeline. So if people want to create a Docker container they need to copy them as well and make sure they are on the classpath btw on the same location as the manifest prescribes. This makes it a little inflexible I would say

Reply:
You may save downloading of non-changing jars and their copying by reusing workspaces. But I see your point with the docker inflexibility. Thanks!